### PR TITLE
Run as user logstash

### PR DESCRIPTION
**Pull request by md5**

### CODE REVIEW

**Comment by daghack** [comment]
LGTM

**Comment by yosifkit** [comment]
LGTM

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Add logstash as command if needed
+if [[ "$1" == -* ]]; then
+	set -- logstash "$@"
+fi
+
+# Run as user "logstash" if the command is "logstash"
+if [[ "$1" == logstash ]]; then
```
Comment (tianon) [inline docker-entrypoint.sh]: Could we skip the special behavior of `[[` here and just use `[ "$1" = logstash ]` instead? :+1:

Comment (md5) [inline docker-entrypoint.sh]: Sure. Anything else before I push the update?

Comment (tianon) [inline docker-entrypoint.sh]: Nope, that's it; the rest looks great. :heart:

**Comment by md5** [comment]
@tianon check it :mag_right:

**Comment by yosifkit** [comment]
No `set -e`? :disappointed:

**Comment by md5** [comment]
:anguished:

**Comment by tianon** [comment]
LGTM :heart:

**Pull request merged**